packages/eval/harbor/test_egress_filter_live.py

```
@@ -0,0 +1,428 @@
+            header = header[: header.index(b"\r\n\r\n") + 4]
+            if not payload:
+                return header, leftover
+            try:
+                sock.sendall(payload)
+            except OSError:
+                return header, leftover
+            return header, leftover + cls._recv_until_close(sock)
+
+    @classmethod
+    def audit_records(cls) -> list[dict[str, object]]:
+        listed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)